server/graphql/invoice/src/mutations/outbound_return/delete.rs

```diff
@@ -60,3 +75,27 @@ pub enum DeleteErrorInterface {
     CannotEditInvoice(CannotEditInvoice),
     CannotDeleteInvoiceWithLines(CannotDeleteInvoiceWithLines),
 }
+
+fn map_error(error: ServiceError) -> Result<DeleteErrorInterface> {
+    use StandardGraphqlError::*;
+    let formatted_error = format!("{:#?}", error);
+
+    let graphql_error = match error {
+        // Structured Errors
+        ServiceError::InvoiceDoesNotExist => {
+            return Ok(DeleteErrorInterface::RecordNotFound(RecordNotFound {}))
+        }
+        ServiceError::CannotEditFinalised => {
+            return Ok(DeleteErrorInterface::CannotEditInvoice(
+                CannotEditInvoice {},
+            ))
+        }
```

Comment:
Are these structured or considered standard errors now?

Reply:
If your question is is this implementation using structured errors then no, this would be `standard` errors.
If you mean should it be a structured error, I don't personally thing so, if the frontend needs to do something special to handle the error then maybe it should be structured...

Reply:
I think both of those are standard 🤷